src/edu/najah/cap/data/data_exporting/helpers/LoggerConfig.java

```diff
@@ -0,0 +1,28 @@
+package edu.najah.cap.data.data_exporting.helpers;
+
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.joran.JoranConfigurator;
+import ch.qos.logback.core.util.StatusPrinter;
+import org.slf4j.LoggerFactory;
+
+public class LoggerConfig {
+    /**
+     * Configures the logger using the Logback framework.
+     * The configuration file is expected to be in XML format.
+     *
+     */
+    public static void setLoggerConfig(){
+        LoggerContext context = (LoggerContext) LoggerFactory.getILoggerFactory();
+
+        try {
+            JoranConfigurator configurator = new JoranConfigurator();
+            configurator.setContext(context);
+            context.reset();
+            configurator.doConfigure("logback.xml");
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
you need to do more action such as exit the program to avoid running the logger with invalid logger